.Rbuildignore

```diff
@@ -17,3 +17,4 @@
 ^revdep$
 ^appveyor\.yml$
 ^\.github$
+^fake-index.html$
```

Comment:
this is needed to have the preview build a site similar to the actual pkgdown website, where dev is at domain.html/dev.